massa-final-state/src/executed_ops.rs

```diff
@@ -22,39 +25,66 @@ use std::ops::Bound::{Excluded, Included};
 
 /// A structure to list and prune previously executed operations
 #[derive(Debug, Default, Clone, PartialEq, Eq)]
-pub struct ExecutedOps(PreHashMap<OperationId, Slot>);
+pub struct ExecutedOps {
+    /// Map of the executed operations
+    ops: PreHashMap<OperationId, Slot>,
+    /// Cumulated hash of the executed operations
+    pub hash: Option<Hash>,
```

Comment:
Why having an option if better then having a default hash from empty data for example or filled of 0 ?

Reply:
Because `ExecutedOps` is created multiple times (in comparison with `Ledger` that is created only once), I'm suspect that having an initial hash in this case would result in different hashes between two nodes with the same executed ops list.